F/Faiss/Faiss_GPU/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+const YGGDRASIL_DIR = "../../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
+
+name = "Faiss_GPU"
```

Comment:
@maleadt Does it make more sense to name packages `_CUDA`, rather than `_GPU`?

There is also some level of AMD rocm support in faiss.

Reply:
How is multiple GPU backends implemented in this package? One advantage to splitting AMD and CUDA into separate packages would be to avoid downloading both sets of dependencies when only one would be needed.